### PR TITLE
Added the Chips Component

### LINKED ISSUE #357: [FEATURE] Chip component
Add a new Chip component, following design and CSS styling as shown in [Bootstrap Italia documentation page](https://italia.github.io/bootstrap-italia/docs/componenti/chips/).

### PR DESCRIPTION
**Pull request by sanyamdogra**
Fixes #357 

#### PR Checklist

- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Added the Chips Component

#### Changes proposed in this pull request:

- Added Chips Component `<Chip>` and `<ChipLabel>`.
- Added the Documentation

### CODE REVIEW

File: src/components/Chips/Chip.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  disabled: false
+}
+
+const Chip = props => {
+  const { className, tag: Tag, simple, large, disabled, ...attributes } = props
+  const wrapperClasses = classNames(
+    className,
+    simple ? 'chip-simple' : false,
+    large ? 'chip-lg' : false,
+    disabled ? 'chip-disabled' : false,
```
Comment (dej611) [inline src/components/Chips/Chip.js]: could you use the object version for this? Having this multiple `false` makes it harder to read.

Something like:
```js
const wrapperClasses = classNames(
  'chip',
  className,
  {
    'chip-simple': simple,
    'chip-lg': large,
    'chip-disabled': disabled
  }
);
```

Comment (dej611) [reply src/components/Chips/Chip.js]: Why then `wrapperClasses` as variable name? Maybe `classes` here is enough.

File: src/components/Chips/ChipLabel.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  className: PropTypes.string
+}
+
+const defaultProps = {
+  tag: 'span'
+}
+
+const ChipLabel = props => {
+  const { className, tag: Tag, ...attributes } = props
+  const classes = classNames('chip-label')
```
Comment (dej611) [inline src/components/Chips/ChipLabel.js]: you forgot to pass `className` here

File: stories/Chips/Chips.stories.js [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+import { Chip, ChipLabel, Button, Icon } from '../../src'
+
+import VersioneStandard from './docs/VersioneStandard.md'
+import GreatVersion from './docs/GreatVersion.md'
+import DisabledChip from './docs/DisabledChip.md'
+import ChipGroups from './docs/ChipGroups.md'
+
+const VersioneStandardComponent = () => (
+  <div className="row">
```
Comment (dej611) [inline stories/Chips/Chips.stories.js]: Can you use the `Row` reactstrap component here?
ref: https://reactstrap.github.io/components/layout/

File: stories/Chips/Chips.stories.js [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+    <div className="col-12 col-md-6">
```
Comment (dej611) [inline stories/Chips/Chips.stories.js]: Can you use the `Col` reactstrap component here?
ref: https://reactstrap.github.io/components/layout/

File: src/components/Chips/Chip.js (hunk 1 above)
Comment (sanyamdogra) [reply src/components/Chips/Chip.js]: Done

File: src/components/Chips/ChipLabel.js (hunk 2 above)
Comment (sanyamdogra) [reply src/components/Chips/ChipLabel.js]: Oops!

File: stories/Chips/Chips.stories.js (hunk 3 above)
Comment (sanyamdogra) [reply stories/Chips/Chips.stories.js]: Sure!

File: stories/Chips/Chips.stories.js (hunk 4 above)
Comment (sanyamdogra) [reply stories/Chips/Chips.stories.js]: Sure!

File: src/components/Chips/Chip.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+const defaultProps = {
+  tag: 'div',
+  simple: false,
+  large: false,
+  disabled: false
+}
+
+const Chip = props => {
+  const { className, tag: Tag, simple, large, disabled, ...attributes } = props
+  const Classes = classNames('chip', className, {
```
Comment (dej611) [inline src/components/Chips/Chip.js]: rename it to `classes`. Capitalized variables are used for components usually.

File: src/components/Chips/ChipLabel.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+const defaultProps = {
+  tag: 'span'
+}
+
+const ChipLabel = props => {
+  const { className, tag: Tag, ...attributes } = props
+  const classes = classNames(className, 'chip-label')
+  return <span {...attributes} className={classes} />
```
Comment (dej611) [inline src/components/Chips/ChipLabel.js]: use `Tag` here instead of `span`.

**Pull request merged**